indico/modules/designer/pdf.py

```diff
@@ -92,22 +94,27 @@ def get_pdf(self):
         return data
 
     def _draw_item(self, canvas, item, tpl_data, content, margin_x, margin_y):
+        override = values_from_signal(signals.event.designer.customize_badge_style.send(item), as_list=True)
```

Comment:
Wouldn't it be better to put the original data in a dict, thne pass is to the signal, and then use that dict, updated with what the signal returned, to get the data for assigning it to `style.whatever`?

ie somehing like this

```
        styles = {alignment: ..., text_color: item.get(...), ...}
        for update in values_from_signal(
            signals.event.designer.customize_badge_style.send(item, styles=styles),
            as_list=True
        ):
            styles.update(update)
```